mlir/include/Mitigation/IR/MitigationOps.td

```diff
@@ -15,13 +15,28 @@
 #ifndef MITIGATION_OPS
 #define MITIGATION_OPS
 
+include "mlir/IR/EnumAttr.td"
+include "mlir/IR/OpBase.td"
 include "mlir/Interfaces/CallInterfaces.td"
 include "mlir/IR/SymbolInterfaces.td"
 include "mlir/IR/BuiltinAttributes.td"
-include "mlir/IR/OpBase.td"
 
```

Comment:
Minor, but it would be nice to keep these sorted (at least roughly in terms of grouping like `IR` and `Interfaces`, unfortunately we don't have a tool that does this for tablegen files).